src/app/store/slices/storage/storage.thunks/uploadFolderThunk.ts

```diff
@@ -153,7 +145,10 @@ export const uploadFolderThunk = createAsyncThunk<void, UploadFolderThunkPayload
         //Added wait in order to allow enough time for the server to create the folder
         await wait(500);
 
-        rootFolderItem = createdFolder;
+        if (!rootFolderItem) {
+          rootFolderItem = createdFolder;
```

Comment:
I don't see the point of this, every time a folder is created in the upload, the rootFolder is replaced. In production if it works to delete the folder when the upload is cancelled, it is possible that it does not identify that this change solves. Can you explain what I'm missing?